lib/Minify/Source/Factory.php

```diff
@@ -110,6 +110,20 @@ public function checkIsFile($file)
         return $realpath;
     }
 
+
+    /**
+     * @param string $path
+     * @return string
+     */
+    public function getNormalizedPath($path)
```

Comment:
Please make this private, replace the line comments with a summary in the phpdoc, and for the $path specify that this only handles absolute file paths.